apps/staking/src/components/node/NodeHireNodeSection.tsx

```diff
@@ -63,6 +63,8 @@ export const NodeHireNodeSection: FC<NodeHireNodeSectionProps> = (props) => {
             px={{ base: 2, lg: 4, xl: 8 }}
             py={{ base: 2, sm: 4, lg: 8 }}
             mb={6}
+            borderRadius={'2xl'}
+            border="1px solid rgba(255, 255, 255, 0.10)"
```

Comment:
I think we have a color var for `rgba(255, 255, 255, 0.10)`. Please check the colors config.